sdk/keyvault/keyvault-secrets/README.md

```diff
@@ -204,8 +204,8 @@ const secretName = "MySecretName";
 async function main() {
   const latestSecret = await client.getSecret(secretName);
   console.log(`Latest version of the secret ${secretName}: `, latestSecret);
-  const specificSecret = await client.getSecret(secretName, { version: latestSecret.version! });
-  console.log(`The secret ${secretName} at the version ${latestSecret.version!}: `, specificSecret);
+  const specificSecret = await client.getSecret(secretName, { version: latestSecret.properties.version! });
```

Comment:
Why do we have to assert using "!" on these lines? That feels messy for a README sample.

Reply:
secrets do not have versions for a bit I believe. They can be received as pending right after they're created.